platform/podjs/package.json

```diff
@@ -21,7 +21,8 @@
     "@zip.js/zip.js": "2.3.7",
     "fp-ts": "^2.8.2",
     "io-ts": "^2.2.16",
-    "rdf-js": "^4.0.2"
+    "rdf-js": "^4.0.2",
+    "rdf-string": "^1.6.0"
```

Comment:
It appears the contents of that file could be easily generated automatically. What is the purpose of having it in the repository? FWIW, it seems somewhat inconsistent with the packages we are using already.

Reply:
It's a long story, but the purpose is basically that we show this to the user on the iOS/Android polyPod apps, similar to `about:license` in Firefox. Since we don't use _podjs_ on Android or iOS, it's not legally necessary to add its dependencies, so I think we can also revisit the whole thing a bit later, see what we're missing and what's been removed in the mean time, maybe actually automate it...

Reply:
@snoack we've tried to be careful about any new dependency that's added to production code, as well as the ones that have been eliminated, but we might have overlooked something. Please create a Jira ticket for the inconsistencies you've observed.